### DISCUSSION TITLE
Github action limits and moving existing repos?

### DISCUSSION THREAD

**Discussion by angrave** [uillinois-community/uillinois-community.github.io#51] [General] [answered]
Hi, what are the github action limits for Illinois github orgs, and what do you recommend for the following?

The context: I have a github repo that should probably be moved inside Illinois-owned github org. Currently it's just a 'regular' github repo but I've just exceeded the storage limits (0.5GB) by using github actions to build Windows+OSX+Linux electron app images. 
I'm wondering what to do, and if moving it under an Illinois Github Org would overcome this limitation.

If moving it is correct, is there an appropriate Illinois owned github org that already exists, or do I need to request a new one?

**Comment by jtgorman** [comment, marked as answer]
From my memory it's kind of fuzzy.  I'm fairly certain the storage is bigger than that by default for most orgs. Some resources are capped per org but can be increased, others just draw from the general pool that the whole University license has.  I'm going to send this on to the AITS team at githubsupport@uillinois.edu.

My gut feeling is moving is probably a good idea. Without knowing a bit more about the project it's kind of hard to say if there's an appropriate org already or . I know there's several departments and colleges (Engineering for sure) that have their own, as well as some general initiatives like Illinois Cloud code or the open source one https://illinois.github.io/ .



Mind giving some more info?

**Reply by angrave** [reply]
It's an electron app that I (and others including CS students) are working for accessibility purposes. It can be used to send live captions and audio descriptions including stream to Zoom and 608 broadcast caption devices that listen on a TCP port. The repo is currently private but we may open it in the future.

**Reply by angrave** [reply]
Update: After some suggestions from Dave Mussulman,I intend to move it into the Illinois-owned org,
https://github.com/grainger-illinois

**Reply by jtgorman** [reply]
Ah, good. One heads up is that I've seen transfers can take a while and they can also fail silently. (It'll look like it works, then at some point you realize that it never actually moved. 

The main gotcha is the person transferring has permissions to create a repository in the target org. I think early on one of our transfers was to an org I didn't have that ability (I think got left off by accident, but it's also possible it was to one of our other orgs that I'm not actually a developer in )

That being said, once we've got over that hurdle it's worked pretty well. You go into settings into the "danger zone" in order to do it.
